packages/baseclient-components/src/component/Menu/FeatureInfo/FeatureInfo.tsx

```diff
@@ -0,0 +1,407 @@
+import * as React from 'react';
+
+import OlMap from 'ol/Map';
+import OlSourceVector from 'ol/source/Vector';
+import OlLayerVector from 'ol/layer/Vector';
+import OlStyleStyle from 'ol/style/Style';
+import OlStyleFill from 'ol/style/Fill';
+import OlStyleStroke from 'ol/style/Stroke';
+import OlStyleCircle from 'ol/style/Circle';
+
+import {
+  Menu
+} from 'antd';
+const MenuItem = Menu.Item;
+
+import {
+  Window,
+  SimpleButton
+} from '@terrestris/react-geo';
+
+const _isEqual = require('lodash/isEqual');
+const _isEmpty = require('lodash/isEmpty');
+
+import MapUtil from '@terrestris/ol-util/dist/MapUtil/MapUtil';
+import FeatureInfoGrid from '../../FeatureInfoGrid/FeatureInfoGrid';
+import './FeatureInfo.less';
+
+interface DefaultFeatureInfoProps {
+  /**
+   * The maximum number of menu items to display.
+   * @type {Number}
+   */
+  maxMenuItems: number
+}
+
+interface FeatureInfoProps extends Partial<DefaultFeatureInfoProps> {
+  /**
+  * The features to render in the menu.
+  * @type {Array}
+   */
+  features: any,
+
+  /**
+   * The ol map.
+   * @type {ol.Map}
+   */
+  map: any, // OlMap
+
+  /**
+   * Translate function
+   */
+  t: (arg: string) => void,
+
+  /**
+   * Dispatch function
+   */
+  dispatch: (arg: any) => void
+}
+
+interface FeatureInfoState {
+  menuHidden: boolean,
+  gridWinHidden: boolean,
+  featuresToShow: any[], //OlFeature[]
```

Comment:
```suggestion
  featuresToShow: any[],
```

Reply:
I'd like to leave this comment here as reminder to change the type after typescript based `ol6` version is released and we finally can use "real" types for openlayers objects.

There are much more similar comments across the code btw.